discord/integrations.py

```diff
@@ -69,28 +106,35 @@ class Integration:
         The integration type (i.e. Twitch).
     enabled: :class:`bool`
         Whether the integration is currently enabled.
-    syncing: :class:`bool`
+    syncing: Optional[:class:`bool`]
         Where the integration is currently syncing.
-    role: :class:`Role`
+    role: Optional[:class:`Role`]
```

Comment:
I'm not sure why the types for these have changed.

Reply:
The changelog mentioned not all fields are necessarily provided anymore. I checked [here](https://discord.com/developers/docs/resources/guild#integration-object) and the appropriate fields were marked as optional (as the code already accepted them being None)